src/pe_source/dnstwistscript.py

```diff
@@ -0,0 +1,261 @@
+"""Use DNS twist to fuzz domain names and cross check with a blacklist."""
```

Comment:
Use consistent language throughout:
```suggestion
"""Use dnstwist to fuzz domain names and cross check with a blocklist."""
```